pkgs/stdenv/darwin/make-bootstrap-tools.nix

```diff
@@ -284,16 +321,15 @@ rec {
         ${stdenv.cc.libc_dev}/lib/system \
         libSystem-boot
 
-      substituteInPlace libSystem-boot/libSystem.B.tbd \
-        --replace "/usr/lib/system/" "$PWD/libSystem-boot/system/"
+      sed -i "s|/usr/lib/system/|$PWD/libSystem-boot/system/|g" libSystem-boot/libSystem.B.tbd
```

Comment:
Is there a reason for preferring sed here?

Reply:
so i moved from `stdenv.mkDerivation` to a plain `derivation` to not get a polluted environment to test and there is not a substitueInPlace available in `derivation`.